Manifests/ManagedPreferencesApplications/com.google.Chrome.plist

```diff
@@ -9753,6 +9849,5 @@ If the policy DeveloperToolsAvailability is set, the value of the policy Develop
 	<key>pfm_unique</key>
 	<true/>
 	<key>pfm_version</key>
-	<integer>14</integer>
-</dict>
```

Comment:
Dictionary closing tag lost.

Reply:
Fixed, sorry not sure how this happened.